src/hazardous/hash/sha512.rs

```diff
@@ -389,7 +389,7 @@ impl Sha512 {
 	}
 }
 
-#[must_use]
+#[must_use = "SECURITY WARNING: Ignoring a Result can have real security implications."]
 /// Initialize a `Sha512` struct.
 pub fn init() -> Sha512 {
```

Comment:
This function does not return a `Result`, so the use of `#[must_use]` should be removed.